src/components/authentication/Login.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react'
+import axios from 'axios'
+import { Link } from 'react-router-dom'
+import { Formik, Form } from 'formik'
+
+import Button from '@material-ui/core/Button'
+import CssBaseline from '@material-ui/core/CssBaseline'
+import TextField from '@material-ui/core/TextField'
+import Grid from '@material-ui/core/Grid'
+import Typography from '@material-ui/core/Typography'
+import { makeStyles } from '@material-ui/core/styles'
+import Container from '@material-ui/core/Container'
+
+const useStyles = makeStyles(theme => ({
+	paper: {
+		marginTop: theme.spacing(8),
+		display: 'flex',
+		flexDirection: 'column',
+		alignItems: 'center'
+	},
+	avatar: {
+		margin: theme.spacing(1),
+		backgroundColor: theme.palette.secondary.main
+	},
+	form: {
+		width: '100%',
+		marginTop: theme.spacing(1)
+	},
+	submit: {
+		margin: theme.spacing(3, 0, 2),
+		borderRadius: 20
+	}
+}))
+
+const Login = () => {
+	const classes = useStyles()
+
+	return (
+		<Container component='main' maxWidth='xs'>
+			<CssBaseline />
+			<div className={classes.paper}>
+				<Typography component='h1' variant='h5'>
+					Sign in
+				</Typography>
+				<form className={classes.form} noValidate>
+					<TextField
+						variant='outlined'
+						margin='normal'
+						required
+						fullWidth
+						id='email'
+						label='Email Address'
+						name='email'
+						autoComplete='email'
+						autoFocus
+					/>
+					<TextField
+						variant='outlined'
+						margin='normal'
+						required
+						fullWidth
+						name='password'
+						label='Password'
+						type='password'
+						id='password'
+						autoComplete='current-password'
+					/>
+					<Button
+						type='submit'
+						fullWidth
+						variant='contained'
+						color='primary'
+						className={classes.submit}
+					>
+						Sign In
+					</Button>
+					<Grid container justify='center'>
+						{/* <Grid item xs>
+							<Link style={{ fontSize: 14, color: '#0000EE' }}>
```

Comment:
We need to get rid of commented out code before we continue with this

Reply:
Yes, Login is still half baked.